scanner_test.go

```diff
@@ -1,16 +1,18 @@
-package sse
+package sse_test
```

Comment:
Please stop doing this. Your tests should be in the same package.

Reply:
@jadr2ddude why should the tests be in the same package? (I thought that is was better to use a `*_test` package)